### ISSUE TITLE
seppop changes levels order in @loc.fac

### ISSUE THREAD

**Issue by thibautjombart** [thibautjombart/adegenet#25]

**Comment by thibautjombart** [comment]
```r
 ## Let's make some cattle hybrids
     ##
     data(microbov)
     
     ## first, isolate each breed
     temp <- seppop(microbov)
     names(temp)
     
     salers <- temp$Salers
     zebu <- temp$Zebu
     borgou <- temp$Borgou
     somba <- temp$Somba
     
  > locNames(salers)
 [1] "INRA63"  "INRA5"   "ETH225"  "ILSTS5"  "HEL5"    "HEL1"    "INRA35" 
 [8] "ETH152"  "INRA23"  "ETH10"   "HEL9"    "CSSM66"  "INRA32"  "ETH3"   
[15] "BM2113"  "BM1824"  "HEL13"   "INRA37"  "BM1818"  "ILSTS6"  "MM12"   
[22] "CSRM60"  "ETH185"  "HAUT24"  "HAUT27"  "TGLA227" "TGLA126" "TGLA122"
[29] "TGLA53"  "SPS115" 
> levels(salers@loc.fac)
 [1] "BM1818"  "BM1824"  "BM2113"  "CSRM60"  "CSSM66"  "ETH10"   "ETH152" 
 [8] "ETH185"  "ETH225"  "ETH3"    "HAUT24"  "HAUT27"  "HEL1"    "HEL13"  
[15] "HEL5"    "HEL9"    "ILSTS5"  "ILSTS6"  "INRA23"  "INRA32"  "INRA35" 
[22] "INRA37"  "INRA5"   "INRA63"  "MM12"    "SPS115"  "TGLA122" "TGLA126"
[29] "TGLA227" "TGLA53" 
>
```

**Comment by zkamvar** [comment]
Ugh. I've seen this come up many times. Basically, you have to manually set the levels for the factors, or it goes automatically to alphabetical. I believe it's one of the circles in the R inferno. I believe the solution would be something along the lines of:

```r
levels(object@loc.fac) <- locNames(object)
```

**Comment by thibautjombart** [comment]
I know. Actually I now force it creating the factor, e.g.
x= factor(rep(a,ntimes), levels=a)

Fixed as of 38b7ce4f4991b6

**Comment by thibautjombart** [comment]
BTW now I'm using tab(x, freq=TRUE) to get a table of allele frequencies
from genind or genpop objects; makefreq is just a wrapper for this.
